binderhub/_version.py

```diff
@@ -1,15 +1,11 @@
-
```

Comment:
Can we exclude this file from these tools? It is copied directly from the versioneer project. This means if we make changes to it it becomes harder to diff our version to a new version we might one day want to upgrade to.

Reply:
I agree, will do.

Reply:
Done!